netbox/dcim/models/device_components.py

```diff
@@ -725,6 +725,13 @@ class Interface(ModularComponentModel, BaseInterface, CabledObjectModel, PathEnd
         object_id_field='assigned_object_id',
         related_query_name='interface',
     )
+    vlan_translation_policy = models.ForeignKey(
```

Comment:
This can probably be moved to BaseInterface as we intend to add it to both Interface and VMInterface.